docs/Coiote_IoT_DM/Coiote_videos/What_collected_IoT_data_can_you_find_in_the_Coiote.md

```diff
@@ -0,0 +1,29 @@
+# What collected IoT data can you find in the Coiote?
```

Comment:
Sound weird...
```suggestion
# What IoT data is collected and stored in the Coiote?
```
or `What types of IoT data is collected and stored in the Coiote?`

Reply:
The filename should be changed too, I think.